packages/blade-old/src/molecules/Amount/Amount.native.js

```diff
@@ -46,6 +46,8 @@ const styles = {
     },
     lineHeight({ size }) {
       switch (size) {
+        case 'xsmall':
+          return 'small';
```

Comment:
@parichay28 You'll have to add changeset for a release to happen

Reply:
@dhruvdutt can you elaborate more, I thought change sets are added by changeset bot

Reply:
Changesets have to be added manually. Try clicking on this once.
![image](https://user-images.githubusercontent.com/5961873/119783602-fd866480-beea-11eb-8caf-f14807cfb8a3.png)

Reply:
@dhruvdutt @sumit-gupta91 changeset is added